test/jobs/sync_beer_catalog_job_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Could you delete this file?

Reply:
mmm, We have to add some tests, but for now ok will delete it